### PR TITLE
Add rpc_url

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/hoo/info/info.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
 {
     "name": "Hoo Token",
     "type": "coin",
+    "type_coin": "0x80000492",
```
Comment (vcoolish) [inline blockchains/hoo/info/info.json:4]:
```suggestion
    "coin_type": 1170,
```

**Comment by vcoolish** [comment]
remove quotes from coint_type value, should be int

File: blockchains/dogechain/info/info.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "website": "https://dogechain.dog/",
     "description": "Dogechain supercharges $Dogecoin to bring crypto applications like NFTs, games, and DeFi to the $Dogecoin community. Unfortunately, $Dogecoin cannot support any of these applications.",
     "explorer": "https://explorer.dogechain.dog",
+    "rpc_url": "https://rpc-us.dogechain.dog",
```
Comment (vcoolish) [inline blockchains/dogechain/info/info.json:9]: remove dogechain folder for now

**Review by vcoolish — APPROVED** [review]

**Comment by hazem454** [comment]
These are my assets and I did not put them for trade or profit, I put them only in order to save my daughter's life and today you are following this

**Review by Ramfel2021 — COMMENTED** [review]
Boo!

**Review by Ramfel2021 — COMMENTED** [review]
Now, claim time [royalties](its just fair and go)

**Pull request merged**